react-app/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
I would recommend to employ CSS in JS approach to style your components for performance and scalability reasons. Even for such a basic app, you may introduce `styled-components` to get your self acquainted with smoother development patterns.